README.md

```diff
@@ -4,4 +4,7 @@
 
 These crates may be used to inspect and validate the internal state of Filecoin Actors.
 
-For the smart-contracts, see: https://github.com/filecoin-project/builtin-actors
+For the smart-contracts, see: https://github.com/filecoin-project/builtin-actors  
+NOTE: Some of the code is copy-pasted from the above repository. Therefore, it's advised to avoid
```

Comment:
Let's be more precise - code in `actors/` and `fil_actors_shared/` _is_ copy pasted. Code in `fil_actor_interface/` is not, and it's okay to tinker around there.